src/java/org/apache/cassandra/index/sai/disk/PostingList.java

```diff
@@ -70,56 +70,7 @@ default PeekablePostingList peekable()
         return new PeekablePostingList(this);
     }
 
-    class DeferredPostingList implements PostingList
```

Comment:
It's not used in CC.  Mike A says, "we got rid of all the deferral because it became obvious that we were using the deferred iterators immediately after they were created so they were instantiated straight away."